index.html

```diff
@@ -1258,6 +1258,52 @@ <h1>Core Properties</h1>
 identifiers.
     </p>
 
+    <p>
+For reference, the core properties found at the top level of a <a>DID
+document</a> are as follows. Properties belonging to other objects referenced
+in the <a>DID document</a> are also listed, with their respective top-level
+property.
+    </p>
+    <ul id="core-properties-summary">
+      <li>
+<code><a>id</a></code>: defined in <a href="#did-subject"></a>
+      </li>
+      <li>
+<code><a>controller</a></code>: defined in <a href="#control"></a>
+      </li>
+      <li>
+<code><a>verificationMethod</a></code>: defined in
+<a href="#verification-methods"></a>. Sub-properties include <code>id</code>,
+<code>type</code>, <code>controller</code>.
+      </li>
+      <li>
+<code><a>authentication</a></code>: defined in <a href="#authentication"></a>.
+      </li>
+      <li>
+<code><a>assertionMethod</a></code>: defined in <a href="#assertionmethod"></a>.
+      </li>
+      <li>
+<code><a>keyAgreement</a></code>: defined in <a href="#keyagreement"></a>.
+      </li>
+      <li>
+<code><a>capabilityDelegation</a></code>: defined in <a href="#capabilitydelegation"></a>.
+      </li>
+      <li>
+<code><a>capabilityInvocation</a></code>: defined in <a href="#capabilityinvocation"></a>.
+      </li>
+      <li>
+<code><a>service</a></code>: defined in <a href="#service-endpoints"></a>.
+Sub-properties include <code>id</code>, <code>type</code> and
+<code><a>serviceEndpoint</a></code>.
+      </li>
+      <li>
+<code><a>created</a></code>: defined in <a href="#created"></a>.
```

Comment:
did we agree that created and updated go in the did document? I recall strongly objecting to that.

Reply:
We did not agree to put `created` and `updated` in the DID Document. We left it open that some DID Methods may choose to do that, but if they do, it's a statement asserted by the `controller` of said document.